packages/components/ovh-shell/src/client/standalone-shell-client.ts

```diff
@@ -0,0 +1,34 @@
+import { initShell } from '../shell';
+import Shell from '../shell/shell';
+import ShellClient from './shell-client';
+import DirectClientMessageBus from '../message-bus/direct-client';
+import plugin from '../plugin';
+
+export default class StandaloneShellClient extends ShellClient {
+  private applicationId: string;
+
+  private shell: Shell;
+
+  constructor(applicationId: string) {
+    const clientMessageBus = new DirectClientMessageBus();
+    const shellMessageBus = new DirectClientMessageBus();
+    super(clientMessageBus);
+    this.applicationId = applicationId;
+    this.shell = new Shell(shellMessageBus);
+    clientMessageBus.addPeer(shellMessageBus);
+    shellMessageBus.addPeer(clientMessageBus);
+  }
+
+  init(): Promise<ShellClient> {
+    return initShell(this.shell).then((shellApi) => {
+      // @TODO set universe here when 2API configuration is patched
+      // const environment = shellApi.getEnvironment();
+      // environment.setUniverse();
+      shellApi.registerPlugin(
+        'i18n',
+        plugin.i18n(this.shell, shellApi.getEnvironment()),
+      );
+      return this;
```

Comment:
Is this `return` necessary ? 

Since this is a class, you already know the `this` before calling it, otherwise, you wouldn't be able to call the init method. 
The method just registers a `Plugin` so it should just be a `Promise<void>`, no ?